src/GraphODataTemplateWriter/CodeHelpers/PHP/TypeHelperPHP.cs

```diff
@@ -80,6 +80,10 @@ public static string GetTypeString(this OdcmType @type)
                 case "Binary":
                 case "Stream":
                     return "\\GuzzleHttp\\Psr7\\Stream";
+                case "Date":
```

Comment:
Remove this to prevent breaking v1 but do a separate PR to `support/php-2.x.x` containing Date changes only. I can merge dev to the v2 branch to get the TimeOfDay and Byte changes